content/chronograf/v1.9/guides/dashboard-template-variables.md

```diff
@@ -360,8 +458,16 @@ _**Example custom meta query variable in a cell query**_
 SELECT "purchases" FROM "animals"."autogen"."customers" WHERE "customer" = :customMetaVar:
 ```
 
-#### Custom meta query variable use cases
-Custom meta query template variables should be used any time you are pulling values from InfluxDB, but the pre-canned template variable types aren't able to return the desired list of values.
+#### InfluxQL meta query variable use cases
+Custom InfluxQL meta query template variables should be used any time you are pulling values from InfluxDB, but the pre-canned template variable types aren't able to return the desired list of values.
```

Comment:
```suggestion
Use custom InfluxQL meta query template variables when predefined template variable types aren't able to return the values you want.
```

Reply:
Curious if it makes sense to provide more detail here...or is it obvious when variables won't return values you want?